build-cmakelibs.sh

```diff
@@ -139,5 +156,13 @@ install -m644 argtable2-13/src/argtable2.h $PS2SDK/ports/include/
 
 build argtable3 -DARGTABLE3_INSTALL_CMAKEDIR="${PS2SDK}/ports/lib/cmake/" -DARGTABLE3_REPLACE_GETOPT=OFF -DARGTABLE3_ENABLE_EXAMPLES=OFF -DARGTABLE3_ENABLE_TESTS=OFF
 
+build libsmb2 
+cp -fr $PS2SDK/ports/lib/libsmb2_ip.a $PS2SDK/ports/lib/libsmb2.a
```

Comment:
Final final final thing... let's fix the static analysis error:
`cp -fr "${PS2SDK}/ports/lib/libsmb2_ip.a" "${PS2SDK}/ports/lib/libsmb2_ips.a"`

And do the same approach with the following lines